source/includes/api/_dataset.md

```diff
@@ -1053,20 +998,151 @@ curl -X POST https://api.resourcewatch.org/v1/dataset/:dataset_id/data-overwrite
 
-## Dataset data sync
 
-To sync the data of a dataset, you need to choose the action type (concat or overwrite), a cron pattern and a valid url. This configuration should be set in the 'sync' property when creating or updating a document dataset.
+#### Errors for overwriting data for a document based dataset
 
-Please be sure that the 'overwrite' property is set to true. This could be used as a lock in order to not allow new updates even if the sync task is actually created.
+Error code     | Error message  | Description
+-------------- | -------------- | --------------
+401            | Dataset is not in saved status   | The dataset's `status` value is not set to `saved`. Refer to [dataset reference](#dataset-reference) for more details on this field and its values.
+401            | Unauthorized   | You need to be logged in to be able to update a dataset.
+403            | Forbidden      | You need to either have the `ADMIN` role, or have role `MANAGER` and be the dataset's owner (through the `userId` field of the dataset).
+403            | Forbidden      | You are trying to update a dataset with one or more `application` values that are not associated with your user account. 
+404            | Endpoint not found  | A dataset with the provided id does not exist, or does not have `connectorType` with value `document`.
+409            | Dataset locked. Overwrite false. | The dataset you're trying to modify has `overwrite` value set to `false`, to explicitly prevent data modifications.
+
+Use this endpoint when:
+
+- Fully replacing the data of an existing document based dataset, while keeping other values like id or name.
+- Modifying the structure of the data for a document based dataset.
+
+
+## Cloning a dataset
+
+> Example request for cloning a dataset
+
+```shell
+curl -X POST https://api.resourcewatch.org/v1/dataset/5306fd54-df71-4e20-8b34-2ff464ab28be/clone \
+-H "Authorization: Bearer <your-token>"
+-H "Content-Type: application/json" -d \
+'{
+  "dataset": {
+    "datasetUrl": "/query/5306fd54-df71-4e20-8b34-2ff464ab28be?sql=select%20%2A%20from%20data%20limit%2010",
+    "application": [
+      "your",
+      "apps"
+    ],
+    "legend": {...},
+    "applicationConfig" : {...}
+  }
+}'
+```
+
+This endpoint allows you to create a new, json based dataset, from the result of a RW API endpoint call. The basic usage example would be to create a new dataset based on a custom query to an existing dataset - this is illustrated in the example `curl` call in this section. Other use cases could be converting the result of an analysis endpoint into a dataset, or capturing the result of a query to a REST based dataset (cartodb, arcgis, etc) to an internal json representation of the same data, that is kept in the API database.   
```

Comment:
Why just json based?

Reply:
This endpoint is meant to create a dataset based on the output of other endpoints of the API - the great majority of which return a JSON. This dataset being a json is basically a way for the API to parse it's own output

Reply:
Okay I think that could use some clarification. Maybe a hypothetical example?

Reply:
let's say you have a dataset with id "12345", called "population per country" with 200 or so rows, one per country. Now let's say we're want to create a new dataset, that only contains the top 5 countries of those 200 or so. You'd use the clone with a body roughly like so:

`{
  "dataset": {
    "datasetUrl": "/query/12345?sql=select * from data order by population desc limit 5",
    "application": [
      "gfw"
    ],
    "legend": {...},
    "applicationConfig" : {...}
  }
}`
'
This would give you a new dataset. Internally, the clone endpoint calls `https://api.resourcewatch.org/query/12345?sql=select * from data order by population desc limit 5` which returns a json, then feeds that to a "create new dataset" POST call. It's JSON because `https://api.resourcewatch.org/query/12345?sql=select * from data order by population desc limit 5` outputs a JSON, like most other API calls.